core/src/main/scala/io/chrisdavenport/github/data/Content.scala

```diff
@@ -175,6 +175,7 @@ object Content {
         "path" -> a.path.asJson,
         "message" -> a.message.asJson,
         "content" -> a.content.asJson,
+        "sha" -> a.author.asJson,
```

Comment:
Probably want the sha and not the author there.

```suggestion
        "sha" -> a.sha.asJson,
```

Reply:
Right! :man_facepalming: